### DISCUSSION TITLE
3.0.9 quickstart package : mysqli_sql_exception Error : HTTP 500 Whoops

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#558] [Q&A] [answered]
Hello Community

I too am trying to install the quick start package.
I read this here:
Just try to install the 3.0.9 quickstart package — Errors and no installation? https://github.com/templaza/astroid-framework/discussions/554
So at least I know I am barking up the right tree.

I have downloaded and unzipped the quickstart package

And have placed it in a local folder on my computer:
///Users/paurraystudio/Sites/localhost/joomla/309-astroid-quickstart-tk2/

In MAMP (on a mac) I created an empty database and called it 309-astroid-quickstart-tk2

In MAMP I created a host called 309-astroid-quickstart-tk2 and pointed it at the folder listed above

I then save in MAMP and with the Host I just created selected use the open in browser function where I get the following errors… Could someone kindly give me a nudge in the right direction?

thanks

Paul

`in /Users/paurraystudio/Sites/localhost/joomla/309-astroid-quickstart-tk2/libraries/vendor/joomla/database/src/Mysqli/MysqliStatement.php (line 138)
in /Users/paurraystudio/Sites/localhost/joomla/309-astroid-quickstart-tk2/libraries/vendor/joomla/database/src/Mysqli/MysqliStatement.php -> prepare (line 138)
in /Users/paurraystudio/Sites/localhost/joomla/309-astroid-quickstart-tk2/libraries/vendor/joomla/database/src/Mysqli/MysqliDriver.php -> __construct (line 1020)
in /Users/paurraystudio/Sites/localhost/joomla/309-astroid-quickstart-tk2/libraries/vendor/joomla/database/src/DatabaseDriver.php -> prepareStatement (line 1785)
DatabaseDriver->setQuery(object(MysqliQuery)) in /Users/paurraystudio/Sites/localhost/joomla/309-astroid-quickstart-tk2/libraries/src/Menu/SiteMenu.php (line 166)

                    )
                    ->bind(':currentDate2', $currentDate)
                    ->order($this->db->quoteName('m.lft'));
                $items    = [];
                $iterator = $this->db->setQuery($query)->getIterator();
                foreach ($iterator as $item) {
                    $items[$item->id] = new MenuItem((array) $item);
                }`

**Comment by deltapapa01** [comment, marked as answer]
Hey,
J5 need  min MySQL 8.
Ist your Wamp maybe on MySql 5.x?

**Comment by paurray** [comment]
Hi @deltapapa01 

You are of course 100% right.
Silly me, MAMP is on 5.7.39.
Much as I really like MAMP they develop stuff at a glacial pace.
https://mampsupportforum.com/viewtopic.php?f=3&t=202&sid=094cb1bca8d8e984cb56019735ba0f64
Might have to start looking at other options.

Thanks for this.

**Comment by Chacapamac** [comment]
Well, @paurray , thats a bummer.
I see that, I’m not sure if that can be a solution?
MySQL 8 or MariaDB with MAMP Pro 6.x —> MySQL 8 or MariaDB with MAMP Pro 6.x

**Comment by paurray** [comment]
@Chacapamac 
I don't know if this is easier but am looking at this now (honestly over my head but will give it a go):
https://www.perplexity.ai/search/easiest-way-to-btJ5MEHRTCuN9GbnIL9Rhw?s=c

**Comment by paurray** [comment]
@Chacapamac 
also following up here: https://mampsupportforum.com/viewtopic.php?p=644#p644
thanks again @@deltapapa01

**Comment by iorbita** [comment]
... the new release of Mamp Pro 7 in near, but the real question is when?
The wait is just too long...
https://x.com/mamp_en/status/1737114424400572698?s=46&t=H8Pjt3U2xw-asdHGNVr9hw

**Comment by paurray** [comment]
@iorbita Yes thanks for this I saw that.It remind me of going to a restaurant and being told you might get a table and then having to wait and wait. Very close to giving up waiting for MAMP here. Looking at alternatives and an still maybe fall back on MAMP later. Fingers crossed. https://github.com/templaza/astroid-framework/discussions/559
